src/app/(marketing)/create-profile/(onboarding)/types.ts

```diff
@@ -1,14 +1,19 @@
+import { Schemas } from "@guildxyz/types"
 import { FunctionComponent } from "react"
 import { SUBSCRIPTIONS } from "./constants"
 
 export type ChainAction = "next" | "previous"
+
 export interface ChainData {
```

Comment:
Why do we call these `chain`? I'd except CreateProfileData or something like that